pylib/mozhginfo/mozhginfo/pushlog_client.py

```diff
@@ -79,38 +82,19 @@ def query_pushid_range(repo_url, start_id, end_id, version=2):
     LOG.debug("About to fetch %s" % url)
     req = requests.get(url)
     pushes = req.json()["pushes"]
-    # pushes.keys() is a list of strings which we need to map to integers
-    # We use reverse in order to return list sorted from newest to oldest push id
-    for push_id in sorted(map(int, pushes.keys()), reverse=True):
+
+    for push_id in sorted(pushes.keys()):
+        print push_id
+        print pushes
         # Querying by push ID is preferred because date ordering is
         # not guaranteed (due to system clock skew)
         # We can interact with self-serve with the 12 char representation
-        revisions.append(pushes[str(push_id)]["changesets"][0])
+        pushs_list.append(Push({push_id: pushes[push_id]}))
```

Comment:
Since we're doing a doing a fetch from pushlog we could then here do the following:
>  pushes_list.append(Push(json_blob=dict(push_id=metadata)))